scipy/stats/_multivariate.py

```diff
@@ -548,9 +548,10 @@ def _cdf(self, x, mean, cov, maxpts, abseps, releps):
 
         """
         lower = np.full(mean.shape, -np.inf)
+        weights = np.ones(1)
         # mvnun expects 1-d arguments, so process points sequentially
-        func1d = lambda x_slice: mvn.mvnun(lower, x_slice, mean, cov,
-                                           maxpts, abseps, releps)[0]
+        func1d = lambda x_slice: mvn.mvnun(lower, x_slice, mean, weights,
```

Comment:
this change should then be undone now that there's a new private `mvnun_weighted`.